datalad/tests/test_handlerepo.py

```diff
@@ -78,9 +78,18 @@ def test_Handle_instance_from_existing(path):
 @with_tempfile
 def test_HandleRepo_instance_brand_new(path):
 
-    gr = HandleRepo(path)
-    assert_is_instance(gr, HandleRepo, "HandleRepo was not created.")
+    annex = AnnexRepo(path)
+    h1 = HandleRepo(path, create=False)
+    assert_is_instance(h1, HandleRepo, "HandleRepo was not created.")
+    assert_false(exists(opj(path, '.datalad')))
+    assert_false(exists(opj(path, '.datalad', 'datalad.ttl')))
+    assert_false(exists(opj(path, '.datalad', 'config.ttl')))
+
+    h2 = HandleRepo(path)
+    assert_is_instance(h2, HandleRepo, "HandleRepo was not created.")
     assert_true(exists(opj(path, '.datalad')))
+    assert_true(exists(opj(path, '.datalad', 'datalad.ttl')))
+    assert_true(exists(opj(path, '.datalad', 'config.ttl')))
```

Comment:
Not yet. Will include it in an updated datalad-metadata.rst

2015-09-02 14:54 GMT+02:00 Yaroslav Halchenko <notifications@github.com>:

> In datalad/tests/test_handlerepo.py
> <https://github.com/datalad/datalad/pull/195#discussion_r38527932>:
>
> >      assert_true(exists(opj(path, '.datalad')))
> > +    assert_true(exists(opj(path, '.datalad', 'datalad.ttl')))
> > +    assert_true(exists(opj(path, '.datalad', 'config.ttl')))
>
> do we have anywhere in docs the description of filesystem hierarchy,
> similar to what you had written on the paper?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/195/files#r38527932>.
>